apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/TempAuthMockTest.java

```diff
@@ -60,8 +72,8 @@ public void testTempAuthRequest() throws Exception {
 
       RecordedRequest auth = tempAuthServer.takeRequest();
       assertEquals(auth.getMethod(), "GET");
-      assertEquals(auth.getHeader("X-Storage-User"), "user");
-      assertEquals(auth.getHeader("X-Storage-Pass"), "password");
+      assertEquals(auth.getHeader(SwiftAuthenticationModule.getIdentityHeaderName()), "user");
+      assertEquals(auth.getHeader(SwiftAuthenticationModule.getIdentityHeaderPass()), "password");
```

Comment:
Once there are two tests, it's better to keep the expected header name hardcoded, to properly verify that the expected configuration is applied.